src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,42 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  width: $calendar-width;
+  gap: $day-gap;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: $day-height;
+    width: $day-width;
+    background-color: #eee;
+    border: $day-border solid black;
+    box-sizing: border-box;
+
+    @for $day from 1 through $days-quantity {
+      &:nth-child(#{$day})::before {
+        content: '#{$day}';
+      }
+    }
+    &:hover {
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+      transition: 0.5s;
```

Comment:
You can move the transition on the element. In this case, you have smooth animation when you hover and unhover.  
Because, now, when I move the cursor outside the day element, the animation cuts.

Also, you need to write which property needs to be animated ( with transition). The default value is 'all' but it has side effects and bad performances.

Reply:
But accordingly to the readme.md I had to set 'all' ((((
![image](https://github.com/mate-academy/layout_calendar/assets/116043993/1fb3f078-e9af-4cff-b58b-d661163a1ad8)

I'm gonna add unhovering effect

Reply:
done

Reply:
@NIKaragu  _All changes should be animated..._ 
It's not about using `all` property. It is about - 0.5 duration for all elements which be animated.